src/ReportPortal.Shared/Internal/Delegating/Exceptions/RetryExecutionException.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace ReportPortal.Shared.Internal.Delegating.Exceptions
+{
+    /// <summary>
+    /// Occurs when retry request execution is unsuccessful.
+    /// </summary>
+    public class RetryExecutionException : AggregateException
+    {
+        private readonly string _message;
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="RetryExecutionException"/>
+        /// </summary>
+        /// <param name="innerExceptions">Inner exceptions.</param>
+        public RetryExecutionException(IEnumerable<Exception> innerExceptions)
+            : this("Request has not finished successfully", innerExceptions)
```

Comment:
`Request has not finished successfully` - let's be more generic here, it not only about http requests.

Reply:
fixed.